src/nvim/mouse.c

```diff
@@ -187,6 +189,8 @@ static void call_click_def_func(StlClickDefinition *click_defs, int col, int whi
   typval_T rettv;
   (void)call_vim_function(click_defs[col].func, ARRAY_SIZE(argv), argv, &rettv);
   tv_clear(&rettv);
+  // Make sure next click does not register as drag when callback absorbs the release event.
```

Comment:
Not sure if there are cases where this matters other than when the callback opens a popupmenu. But this seems like a better that might cover more cases without having to expose `got_click`.

Reply:
This is indeed a better solution as the callback may also be something that calls `getchar()` to get mouse input.